content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -10,29 +10,32 @@ title: Tools for Monitoring Resources
 To scale an application and provide a reliable service, you need to
 understand how the application behaves when it is deployed. You can examine
 application performance in a Kubernetes cluster by examining the containers,
-[pods](/docs/user-guide/pods), [services](/docs/user-guide/services), and
+[pods](/docs/concepts/workloads/pods/),
+[services](/docs/concepts/services-networking/service/), and
 the characteristics of the overall cluster. Kubernetes provides detailed
 information about an application's resource usage at each of these levels.
 This information allows you to evaluate your application's performance and
 where bottlenecks can be removed to improve overall performance.
 
-
-
 <!-- body -->
 
-In Kubernetes, application monitoring does not depend on a single monitoring solution. On new clusters, you can use [resource metrics](#resource-metrics-pipeline) or [full metrics](#full-metrics-pipeline) pipelines to collect monitoring statistics.
+In Kubernetes, application monitoring does not depend on a single monitoring solution.
+On new clusters, you can use [resource metrics](#resource-metrics-pipeline) or
+[full metrics](#full-metrics-pipeline) pipelines to collect monitoring statistics.
 
 ## Resource metrics pipeline
 
 The resource metrics pipeline provides a limited set of metrics related to
-cluster components such as the [Horizontal Pod Autoscaler](/docs/tasks/run-application/horizontal-pod-autoscale) controller, as well as the `kubectl top` utility.
+cluster components such as the
+[Horizontal Pod Autoscaler](/docs/tasks/run-application/horizontal-pod-autoscale/)
+controller, as well as the `kubectl top` utility.
 These  metrics are collected by the lightweight, short-term, in-memory 
 [metrics-server](https://github.com/kubernetes-incubator/metrics-server) and
  are exposed via the `metrics.k8s.io` API. 
 
 metrics-server discovers all nodes on the cluster and 
 queries each node's 
-[kubelet](/docs/reference/command-line-tools-reference/kubelet) for CPU and 
+[kubelet](/docs/reference/command-line-tools-reference/kubelet/) for CPU and 
```

Comment:
This could be a glossary tooltip

Reply:
Can change this to a glossary tooltip, but what is the advantage of such a change?

Reply:
Readers can hover for a reminder of what a kubelet is, saving mental effort vs. loading a link and “paging out” the document they were reading before.

Reply:
Let's move this forward. I would look at the entire page to see if the page overused tooltips (or links).